hstest/stage/plotting_test.py

```diff
@@ -0,0 +1,14 @@
+from hstest.stage.stage_test import StageTest
+from hstest.testing.runner.plot_testing_runner import PlottingTestingRunner
+
+
+class PlottingTest(StageTest):
+
+    def __init__(self, source: str = ''):
+        super().__init__(source)
+        self._drawings = []
+        self.runner = PlottingTestingRunner(self._drawings)
+
+    @property
+    def all_figures(self):
+        return self._drawings
```

Comment:
It would be great to have "new_figures" that will return only new figures that were drawn since the last "new_figures" call. And "all_figures" will return all drawn figures as it is now.